lib/registration.pm

```diff
@@ -145,26 +145,19 @@ sub scc_version {
 Wrapper for SUSEConnect -p $name.
 =cut
 sub add_suseconnect_product {
-    my ($name, $version, $arch, $params, $timeout) = @_;
+    my ($name, $version, $arch, $params, $timeout, $retry) = @_;
     assert_script_run 'source /etc/os-release';
     $version //= '${VERSION_ID}';
     $arch    //= '${CPU}';
     $params  //= '';
-    assert_script_run("SUSEConnect -p $name/$version/$arch $params", $timeout);
-}
-
-=head2 remove_suseconnect_product
-
-    remove_suseconnect_product($name, [$version, [$arch, [$params]]]);
+    $timeout //= 300;
+    $retry   //= 1;                 # run SUSEConnect a 2nd time to workaround the gpg error due to missing repo key on 1st run
 
-Wrapper for SUSEConnect -d $name.
-=cut
-sub remove_suseconnect_product {
```

Comment:
Are you sure you want to remove this method? Seems it's still used

Reply:
no, thanks for review:)